mtp_noms_ops/assets-src/stylesheets/views/_tags.scss

```diff
@@ -0,0 +1,9 @@
+.rejected-tag {
+  color: #d4351c;
```

Comment:
is this a colour from the standard gds palette? there'd be a variable name to use if so